_components/footers.md

```diff
@@ -6,16 +6,244 @@ title: Footers
 
 <div class="preview">
   <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Footer_FullUI_v1-930width.png">
+
+  <h3>Footer Big</h3>
+
+  <footer class="usa-footer usa-footer-big usa-sans" role="contentinfo">
+    <div class="grid-box usa-footer-return-to-top">
+      <a href="#usa-top-of-page">Return to top</a>
+    </div>
+    <div class="usa-footer-primary-section">
+      <div class="grid-box">
+        <nav class="usa-footer-nav width-two-thirds">
+          <ul class="usa-unstyled-list width-one-fourth">
+            <h3 class="usa-footer-primary-link">Topic</h3>
+            <li><a href="javascript:void(0)">Secondary link</a></li>
```

Comment:
This is just for now, the user can replace this with their own link. Using `javascript:void(0)` instead of `#` prevents it from jumping to the top of the page.